readme.md

```diff
@@ -547,10 +565,12 @@ if you wish to create your own store.
 ## Request API
 
 A `request.rateLimit` property is added to all requests with the `limit`,
```

Comment:
I'm fine with full names in our codebase, but perhaps we should rename `request` to `req` in our docs to match Express?

> In this documentation and by convention, the object is always referred to as req (and the HTTP response is res) 

http://expressjs.com/en/4x/api.html#req

Reply:
Ohkay, sure!